frontend/src/home/components/Homepage.jsx

```diff
@@ -25,26 +25,42 @@ export default class Homepage extends Component {
     }
 
     static propTypes = {
-        dispatch: PropTypes.func.isRequired,
         onChangeLocation: PropTypes.func.isRequired,
         showOnboarding: PropTypes.bool.isRequired,
-        user: PropTypes.object.isRequired
+        user: PropTypes.object.isRequired,
+        activity: PropTypes.array,
+        hasSampleDataset: PropTypes.bool,
+        fetchActivity: PropTypes.func.isRequired,
+        fetchRecentViews: PropTypes.func.isRequired,
+        fetchHasSampleDataset: PropTypes.func.isRequired
     };
 
+    async componentDidMount() {
+        try {
+            await this.props.fetchHasSampleDataset();
```

Comment:
What if we simply added this as part of the MetabaseSettings and we include it in the values on window.MetabaseBootstrap?  

Doing that would mean we only have to call `MetabaseSettings.get('sample_data_available')` and you'd have the value you need and you'd never actually have to make a call to list the dbs.